Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -334,7 +322,8 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
 - (CGSize)sizeWithFont:(UIFont*)font constrainedToSize:(CGSize)size lineBreakMode:(UILineBreakMode)lineBreakMode {
     return [self _sizeWithAttributes:@{
         NSFontAttributeName : font,
-        NSParagraphStyleAttributeName : _paragraphStyleWithLineBreakMode(lineBreakMode)
+        static_cast<NSString*>(kCTParagraphStyleAttributeName) :
+            (id)[_paragraphStyleWithLineBreakMode(lineBreakMode) _createCTParagraphStyle]
```

Comment:
I considered that but found it would make more sense to just have synthesized ivars for properties than have to make getters/setters to go through CT methods, and NS has settings that CT doesn't.